pkgs/data/misc/papirus-folders/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "papirus-folders";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner = "PapirusDevelopmentTeam";
+    repo = "papirus-folders";
+    rev = "v${version}";
+    sha256 = "sha256-ZZMEZCWO+qW76eqa+TgxWGVz69VkSCPcttLoCrH7ppY=";
+  };
+
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  meta = with lib; {
+    description = ''
+      papirus-folders is a bash script that allows changing the color of
+      folders in Papirus icon theme and its forks (which based on version 20171007 and newer).
+    '';
```

```suggestion
    longDescription = ''
      papirus-folders is a bash script that allows changing the color of
      folders in Papirus icon theme and its forks (which based on version 20171007 and newer).
    '';
```

`description` should be exactly one short line.

---

corrected